lib/puppetserver/ca/local_certificate_authority.rb

```diff
@@ -82,6 +84,11 @@ def sign_master_cert(int_key, int_cert, csr)
           extension = ef.create_extension(*ext)
           cert.add_extension(extension)
         end
+
+        # Status API access for the CA CLI
+        cli_auth_ext = OpenSSL::X509::Extension.new(CLI_AUTH_EXT_OID, OpenSSL::ASN1::UTF8String.new("true").to_der, false)
```

Comment:
could just do this I think?
```
ef.create_extension(CLI_AUTH_EXT_OID, "true", false)
```

Reply:
Not that its really any better, but its what we do above and below

Reply:
I had run into issues with the factory, but that was probably because I was trying to use an invalid short name. I can try that with the full OID.